.travis.yml

```diff
@@ -6,7 +6,7 @@ addons:
   ssh_known_hosts:
     - github.com
 node_js:
-  - 10
+  - 12.18.4
```

Comment:
We also need to update appveyor https://github.com/bigcommerce/stencil-cli/blob/e0b638709a8e51fb420fc8cca17357381144f01f/appveyor.yml#L3

Reply:
I did it, it should be visible in https://github.com/bigcommerce/stencil-cli/pull/645/files